AdaptiveCards/authoring-cards/input-validation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Input Validation
+author: rebeccaanne
+ms.author: rebecch
+ms.date: 07/24/2020
+ms.topic: article
+---
+
+# Input Validation
+
+Versions 1.3 and later of the Adaptive Cards schema support client side input validation of Input types.
```

Comment:
Nit nit:  ',' after schema

Reply:
I'm not sure a comma goes there? I could do: "Adaptive cards, in versions 1.3 and later of the schema, supports client side input validation of Input types" Does that seem clearer?

Reply:
Should we include a note about guidance and expectations for custom input types (backcompat and newer ones)?

Reply:
> I'm not sure a comma goes there? I could do: "Adaptive cards, in versions 1.3 and later of the schema, supports client side input validation of Input types" Does that seem clearer?

Yeah or maybe even "In versions 1.3 and later of the schema, AdaptiveCards supports ... "

Reply:
> Should we include a note about guidance and expectations for custom input types (backcompat and newer ones)?

Is there something specific you had in mind here?

Reply:
In terms of the Extensibility options [here](https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/extensibility) do things just work as far as any custom inputs they might have had, post-upgrade to 1.3/renderer lib 2.0?

Reply:
Ok, digging in a bit I have some questions myself. I think that's going to have to be beyond the scope of this change. The extensibility documentation in general and for inputs in particular even prior to this work is pretty lacking.  I'm going to log a new issue for input extensibility documentation to follow up.

Reply:
#319